Analytics/EntityStoreTools/Common/ContractValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿// ------------------------------------------------------------------------------
```

Comment:
Isn't this class a duplicate? Looks like you have the same validator in the `Common` package.